src/senselab/text/tasks/emotional_analysis/constants.py

```diff
@@ -0,0 +1,33 @@
+"""This module defines the Emotion enum and related functions for emotional analysis."""
+
+from enum import Enum
+
+
+class Emotion(Enum):
```

Comment:
I am not sure that pre-defining the emotional classes is a good idea. What if someone wants to use a model that can capture an emotion that is not in your pool?

Reply:
you're right, will make changes to the test cases as well.